MonoidalCategories/gap/SymmetricClosedMonoidalCategoriesDerivedMethods.gi

```diff
@@ -907,3 +907,27 @@ AddDerivationToCAP( MonoidalPostComposeMorphismWithGivenObjects,
     
 end : CategoryFilter := IsSymmetricClosedMonoidalCategory and IsStrictMonoidalCategory,
       Description := "MonoidalPostComposeMorphismWithGivenObjects using evaluation, and tensor hom adjunction" );
+
+##
+AddDerivationToCAP( IsomorphismFromDualToInternalHom,
+        
+  function( cat, a )
```

Comment:
I would prefer a diagram for easier reading. That would also be consistent with the other derivations.

Reply:
Very good, this revealed a mistake :)